dev-docs/bidders/zmaticoo.md

```diff
@@ -22,7 +22,7 @@ To use this bidder you will need a valid pubId. For further information, please
 {: .table .table-bordered .table-striped }
 | Name                 | Scope    | Description                                                                                                                | Example                 | Type           |
 |----------------------|----------|----------------------------------------------------------------------------------------------------------------------------|-------------------------|----------------|
-| `pubId`              | required | The pubId provided by zmaticoo                                                                                    |        |`"prebid_test"`          | `string`       |
+| `pubId`              | required | The pubId provided by zmaticoo                                                                                             |`"prebid_test"`          | `string`       |
 | `user`               | required | The object containing user data (See OpenRTB spec)                                                                         | `user: {}`              | `object`       |
```

Comment:
you are not allowed to require the publisher to supply user, device, or other ORTB objects as bidder-specific parameters. Your adapter must read them from the standard location. 

i.e. the publisher does a pbjs.setConfig({ortb2: { device: {...}}}) and your adapter reads them from there.

Reply:
ok